### PR TITLE
Som devel 20

### PR DESCRIPTION
**Pull request by sorgom**

### CODE REVIEW

File: application/components/LCR/README.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
+# component LCR
+**level crossings**
+
+**LCR_Hub.h**
```
Comment (sombeam) [inline application/components/LCR/README.md]: something went wrong here

File: SIL4.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+-Wall
+
+|Flag|Effect|
+|----|----|
+|-pedantic-errors|self explaining|
+|-Werror|tread warnings as errors|
+|-Wall|do not suppress any warning|
+
+## Code coverage
+### Gcov
```
Comment (sombeam) [inline SIL4.md:32]: finish before merge

**Review by sombeam — CHANGES_REQUESTED** [review]

File: application/components/LCR/README.md (hunk 1 above)
Comment (sorgom) [reply application/components/LCR/README.md]: done

File: SIL4.md (hunk 2 above)
Comment (sorgom) [reply SIL4.md:32]: on hold ...

**Review by sombeam — APPROVED** [review]

**Pull request merged**